inc/setup/cachify.hdd.htaccess.php

```diff
@@ -19,18 +19,14 @@
   RewriteCond %{HTTPS} off
   RewriteRule .* - [E=CACHIFY_HOST:%{HTTP_HOST}]
 
-  # set subdirectory
-  RewriteCond %{REQUEST_URI} /$
-  RewriteRule .* - [E=CACHIFY_DIR:%{REQUEST_URI}]
-  RewriteCond %{REQUEST_URI} ^$
-  RewriteRule .* - [E=CACHIFY_DIR:/]
 {{GZIP}}
   # Main Rules
   RewriteCond %{HTTP_ACCEPT} .*text/html.*
+  RewriteCond %{REQUEST_URI} /$
   RewriteCond %{REQUEST_METHOD} GET
-  RewriteCond %{QUERY_STRING} =""
-  RewriteCond %{REQUEST_URI} !^/(wp-admin|wp-content/cache)/.*
-  RewriteCond %{HTTP_COOKIE} !(wp-postpass|wordpress_logged_in|comment_author)_
+  RewriteCond %{QUERY_STRING} ^$
```

Comment:
I don’t have strong feelings about the notation, but is there any benefit in using an empty RegEx over comparison against an empty string?